src/util/sss_krb5.c

```diff
@@ -1375,3 +1375,16 @@ krb5_error_code sss_krb5_init_context(krb5_context *context)
 
     return kerr;
 }
+
+bool sss_krb5_creds_compare(krb5_context kctx, krb5_creds *a, krb5_creds *b)
```

Comment:
The goal is to compare new cred that is being stored with existing creds in a ccache. We remove the matching creds from the ccache and store the new one. This is to keep only unique creds and avoid unnecessary cache growth. @frozencemetery is this sufficient to match the credentials?

Reply:
My memory is that yes, this should be fine.

(Please note that I'm no longer the right person to ask these questions to going forward, as I changed teams at the beginning of this month.)